lib/services/access_request.go

```diff
@@ -2324,15 +2330,15 @@ func (m *RequestValidator) pruneResourceRequestRoles(
 	ctx context.Context,
 	resourceIDs []types.ResourceID,
 	loginHint string,
-	roles []string,
+	requestedRoles []string,
```

Comment:
I sort of disagree with renaming this to `requestedRoles` since nobody has requested them at this point, this is just the full list of candidate roles that it's possible to request. You could name it `candidateRoles` if you prefer